src/nodes.coffee

```diff
@@ -3097,12 +3211,13 @@ exports.If = class If extends Base
 
 UTILITIES =
   modulo: -> 'function(a, b) { return (+a % (b = +b) + b) % b; }'
-
+  objectWithoutKeys: -> 'function(obj, props) { return Object.keys(obj).reduce(function(a,c) { return (![].includes.call(props, c)) && (a[c] = obj[c]), a; }, {}); }'
```

Comment:
Actually, the utility function is [invoked with `o`](https://github.com/GeoffreyBooth/coffeescript/blob/d59be16ab1402fc5965032a93648604d10e2f8a6/src/nodes.coffee#L3246), so this could indeed use the utilities!